ToolBox/GoogleDownloader/GoogleDataDownloader.cs

```diff
@@ -134,5 +230,15 @@ private static DateTime FromUnixTime(long unixTime)
             return epoch.AddSeconds(unixTime);
         }
 
+        private static string ConvertTicker(Symbol symbol)
+        {
+            // FIXME: this is a hack to make sure google finds our index:
+            if (symbol.Value == "SPY")
+            {
+                return "NYSEARCA:SPY";
```

Comment:
I was seeing client.DownloadString failing and not returning any content, but can no longer reproduce this. I have removed this routine for now. Not sure why the patient has healed itself, this may or may not be related to the '-' issue discussed above.